setup.py

```diff
@@ -152,8 +152,8 @@ def get_spec(file_name: str) -> SpecObject:
 SSZObject = TypeVar('SSZObject', bound=View)
 '''
 SUNDRY_CONSTANTS_FUNCTIONS = '''
-def ceillog2(x: uint64) -> int:
-    return (x - 1).bit_length()
+def ceillog2(x: uint64) -> uint64:
+    return uint64((x - 1).bit_length())
```

Comment:
This looks reasonable. Maybe we should do a special condition on the `0` case to avoid underflow

Reply:
If `x` has `uint64` type, then `ValueError` will be thrown, when trying to subtract `1` from `0`.
However, if `0` of `int` type occasionally passed, then the result will be incorrect.
I've added a check that an input should be positive (>= 1) and made `x` have `int` type (e.g. to allow something like `ceillog2(8)`).